### PR TITLE
Add since_seconds filter when reading pod logs

### PR DESCRIPTION
**Pull request by LissaGreense**
Hi! I added option to filter pod logs with since_seconds parameter :D

Before merge following needs to be applied:
- [X] At least one example testcase added in testcases/
- [X] Library Documentation regenerated according to [Generate docs](https://github.com/devopsspiral/KubeLibrary#generate-docs)
- [x] PR entry added in CHANGELOG.md in **In progress** section
- [x] All new testcases tagged as **prerelease** along other tags to exclude it from execution until released on PyPI
- [x] Coverage threshold increased in [.coveragerc](https://github.com/devopsspiral/KubeLibrary/blob/master/.coveragerc) if new coverage is higher than actual, see the lint-and-coverage step in CI
```
fail_under = 86
```

### CODE REVIEW

**Comment by LissaGreense** [comment]
image
There is a problem with CI... :C

File: test/test_KubeLibrary.py [hunk 1]
```diff
@@ -19,6 +19,7 @@ class AttributeDict(object):
     allowing you to recurse down nested dicts (like: AttributeDict.attr.attr)
     """
     def __init__(self, entries):
+        self._root = None
```
Comment (m-wcislo) [inline test/test_KubeLibrary.py:22]: Why do we need it?

**Review by m-wcislo — COMMENTED** [review]
Hi, thanks for the PR. I think it looks ok. One comment about _root only. Also I will try to checkout CI meanwhile.

File: test/test_KubeLibrary.py (hunk 1 above)
Comment (LissaGreense) [reply test/test_KubeLibrary.py:22]: We don't need it, but I think is a good practice to keep instance variables in __init__. Even My linter says it's a code smell :)

Boring stackoverflow argumentation: https://stackoverflow.com/questions/38377276/instance-variables-in-methods-outside-the-constructor-python-why-and-how

Comment (m-wcislo) [reply test/test_KubeLibrary.py:22]: Aaa sorry I was looking at KubeLibrary.py and couldn't fin d it, yeah that's fine.

**Review by m-wcislo — APPROVED** [review]

**Pull request merged**